codalab_service.py

```diff
@@ -738,6 +785,27 @@ def start_services(self):
                 root=(not self.args.codalab_home),
             )
 
+            print_header('Configuring monitoring service')
+            if all(
```

Comment:
Break this up into 4 if statements...if a subset are specified, just set a subset.